recipes/hidapi/all/conandata.yml

```diff
@@ -9,5 +12,5 @@ sources:
     url: "https://github.com/libusb/hidapi/archive/hidapi-0.11.0.tar.gz"
     sha256: "391d8e52f2d6a5cf76e2b0c079cfefe25497ba1d4659131297081fc0cd744632"
   "0.10.1":
-    url: "https://github.com/libusb/hidapi/archive/refs/tags/hidapi-0.10.1.tar.gz"
+    url: "https://github.com/libusb/hidapi/archive/hidapi-0.10.1.tar.gz"
```

Comment:
FYI, refs/tags is less ambiguous (no collision between branch & tags) and it's now the link provided by github for  auto generated archives of tags, so I would advice to not remove these refs/tags, and instead fix other url.